AllReadyApp/Web-App/AllReady/DataAccess/AllReadyDataAccessEF7.ActivitySignup.cs

```diff
@@ -39,24 +39,31 @@ Task IAllReadyDataAccess.AddActivitySignupAsync(ActivitySignup userSignup)
             return _dbContext.SaveChangesAsync();
         }
 
-        Task IAllReadyDataAccess.DeleteActivitySignupAsync(int activitySignupId)
+        Task IAllReadyDataAccess.DeleteActivityAndTaskSignupsAsync(int activitySignupId)
         {
             var activity = _dbContext.ActivitySignup.SingleOrDefault(c => c.Id == activitySignupId);
```

Comment:
Local variable should probably be called `activitySignup` instead of `activity` since that's what it is.